.github/workflows/test.yaml

```diff
@@ -34,6 +34,9 @@ jobs:
     - name: Verify manifest consistency
       run: |
         make verify-manifests
+    - name: Verify modules are tidy
+      run: |
+        make verify-tidy
```

Comment:
Consider adding the `verify-tidy` make target as a dependency of the `lint` target instead of this extra build step.

Reply:
Do you have a good example I can look at? My Makefile skills aren't that strong, and it doesn't look like we have an existing multi-stage target.

Reply:
Sure.

In this line:
https://github.com/Kong/kubernetes-ingress-controller/blob/b240ff231b9fd1b1d91aa0d0d0ebcc87045e4ffa/Makefile#L24

replace `lint:` with `lint: verify-tidy`.

This way, when running `make lint`, `make` will ensure that the `verify-tidy` target is up to date. And since `verify-tidy` is `.PHONY`, it's always out of date. Hence, `make` will always run `verify-tidy` when running `lint`.

Reply:
Done. Rebased garbage PR after and its latest run looks like what we want: https://github.com/Kong/kubernetes-ingress-controller/pull/948/checks?check_run_id=1387299132